parteh/PRTLossFluxesMod.F90

```diff
@@ -0,0 +1,659 @@
+module PRTLossFluxesMod
+
+  use EDPftvarcon,   only : EDPftvarcon_inst
+  use PRTGenericMod, only : prt_vartypes
+  use PRTGenericMod, only : leaf_organ
+  use PRTGenericMod, only : fnrt_organ
+  use PRTGenericMod, only : sapw_organ
+  use PRTGenericMod, only : store_organ
+  use PRTGenericMod, only : repro_organ
+  use PRTGenericMod, only : struct_organ
+  use PRTGenericMod, only : all_carbon_species
+  use PRTGenericMod, only : carbon_species    ! This is a vector
+  use PRTGenericMod, only : carbon12_species
+  use PRTGenericMod, only : nitrogen_species
+  use PRTGenericMod, only : phosphorous_species
+  use PRTGenericMod, only : un_initialized
+  use PRTGenericMod, only : check_initialized
+  use PRTGenericMod, only : num_organ_types
+  use PRTGenericMod, only : prt_global
+  use FatesInterfaceMod, only : hlm_freq_day
+
+  use FatesConstantsMod, only : r8 => fates_r8
+  use FatesConstantsMod, only : i4 => fates_int
+  use FatesConstantsMod, only : nearzero
+  use FatesConstantsMod, only : calloc_abs_error
+  use FatesGlobals     , only : endrun => fates_endrun
+  use FatesGlobals     , only : fates_log 
+  use shr_log_mod      , only : errMsg => shr_log_errMsg
+
+  
+  implicit none
+  private
+
+  ! -------------------------------------------------------------------------------------
+  ! This module hosts the public functions that handle all things
+  ! related to loss fluxes.  They broadly cover the two types of turnover;
+  ! that which happens as events (storms, deciduous drop, herbivory
+  ! fire, etc), and maintenance turnover (constant background) 
+  ! of evergreens, and branchfall).
+  !
+  ! IMPORTANT POINTS! 
+  ! Retranslocation is handled by a single
+  ! flag that defines the mode for each PFT.  So there
+  ! are assumptions here.  A deciduous plant does not
+  ! have maintenance leaf and fine-root turnover, and vice
+  ! versa.  Therefore, the retranslocation parameter
+  ! will have different meanings potentially, for each PFT.
+  ! 
+  ! Branchfall occurs for each PFT (it may be at a reduced rate,
+  ! but it will be called none-the-less).
+  !
+  ! THIS ROUTINE ONLY DEALS WITH LOSSES OF BIOMASS FROM PLANTS THAT ARE SURVIVING
+  ! AN EVENT.  IF A PLANT DIES, THEN THIS ROUTINE DOES NOT HANDLE ITS FLUXES. It
```

Comment:
```suggestion
  ! AN EVENT.  IF A PLANT DIES, THEN THESE ROUTINES DO NOT HANDLE ITS FLUXES. It
```

Reply:
The comments in that part of the code need a good bit more cleaning.  Will address.